lib/npm.js

```diff
@@ -96,6 +102,7 @@ async function getLatest (name) {
 
 module.exports = {
   get: get,
+  post: post,
```

Comment:
let's use the shorthand for this, i know the get isn't in this format but this can be rewritten as 

`
{
    get,
    post,
    getTarball,
    getLatest
}`